src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
please remove this class. you needed it to check yourself

Reply:
sure!